packages/components/src/theme/entities.scss

```diff
@@ -16,7 +16,8 @@
   padding-top: 7px;
 }
 
-.has-error .Select-control, .has-error .form-control, .has-error .form-control:focus {
+.has-error .Select-control, .has-error .select-input__control,
```

Comment:
As I mentioned to you before I would consider just replacing `.Select-control` with `.select-input__control`.

Reply:
done